scripts/build_setup_and_run_navitia_demo.sh

```diff
@@ -0,0 +1,232 @@
+#!/bin/sh
+
+scripts_dir="$(dirname $(readlink -f navitia))"
+ntfs_data_dir=
+osm_file=
+skip_bina=
+kraken_pid=
+jormun_pid=
+
+clean_exit()
+{
+ # kill the background job at the end
+ echo "killing kraken (pid=$kraken_pid) and jormungandr (pid=$jormun_pid)";
+ kill $kraken_pid
+ kill -TERM $jormun_pid
+ exit 0
+}
+
+usage()
+{
+cat << EOF
+usage: $0 options
+
+performs the following actions :
+
+ - build kraken with cmake, make
+ - download osm and ntfs (by default : Paris city in example)
+ - load osm and ntfs into ed bdd
+ - genrate data.nav.lz4 form bdd ed
+ - create basic kraken configuration file and run
+ - create basic jormun configuration files and run
+
+by default, ED database used :
+bdd name = navitia
```

Comment:
:fr: again, if you're willing to do it: ctrl+F `bdd` on all this PR (there are more) :wink: